tsconfig.packages.json

```diff
@@ -154,6 +154,9 @@
     },
     {
       "path": "./modules/web-demo"
+    },
+    {
+      "path": "./modules/sdk-lib-mpc"
     }
```

Comment:
nit: would be nice to keep this list sorted alphabetically.

Reply:
sorted